lib/src/subject_value_wrapper.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:rxdart/rxdart.dart';
```

Comment:
Would it be great to place this file under the `src/model` package?

Reply:
Also moved the HydratedSubject implementation file into `src` package, leaving only an export file in `lib`.